### PR TITLE
feat: create UI components accordion

### PR DESCRIPTION
**Pull request by yuki-92c**
## Overview

- Add accordion component in the components/ui folder based on the design on Figma.

## Changes

- Install checkbox component from shadcn/ui
    - https://ui.shadcn.com/docs/components/accordion
- Style the design

## Review points

- The styling matches the Figma design

## Screenshots or videos

- figma UI component ( landlord )
image

- figma mockup ( tenant )
image

- accordion component 
image

## Notes

- The contents of the accordion differs from landlord and tenant. Tenant's page needs date.
- To display date you have to set `scheduleDate` as a prop.

- sample code
```ts
import {
  Accordion,
  AccordionContent,
  AccordionItem,
  AccordionTrigger,
} from "@/components/ui/accordion"
import { CircleEllipsis, Ellipsis } from "lucide-react"

export default function Home() {
  return (
    <main className="flex min-h-screen flex-col items-center justify-between pt-32">
      <div className="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex">
        <Accordion type="single" collapsible className="w-full">
        <AccordionItem value="item-1">
          <AccordionTrigger scheduleDate={"12/01-12/07"}>Bathroom
          <CircleEllipsis className='ml-4'/> 
          </AccordionTrigger >
          <AccordionContent>
            <div className="flex">
              <div className='items-center'>
                <div className="text-lg font-medium">mop the floor</div>
                <p className="leading-7 text-sm [&:not(:first-child)]:mt-3">
                  The king, seeing how much happier his subjects were, realized the error of
                  his ways and repealed the joke tax.
                </p>
              </div>
              <Ellipsis className='h-4 w-4 shrink-0 transition-transform duration-200 ml-auto' />
            </div>
          </AccordionContent>
          <AccordionContent>
            <div className="flex">
              <div className='items-center'>
                <div className="text-lg font-medium">mop the floor</div>
                <p className="leading-7 text-sm [&:not(:first-child)]:mt-3">
                  The king, seeing how much happier his subjects were, realized the error of
                  his ways and repealed the joke tax.
                </p>
              </div>
              <Ellipsis className='h-4 w-4 shrink-0 transition-transform duration-200 ml-auto' />
            </div>
          </AccordionContent>
        </AccordionItem>
        <AccordionItem value="item-2">
          <AccordionTrigger>Kitchen</AccordionTrigger>
          <AccordionContent>
            Yes. It comes with default styles that matches the other
            components&apos; aesthetic.
          </AccordionContent>
        </AccordionItem>
      </Accordion>
      </div>
    </main>
  );
}

```

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [ ] Code readability and simplicity
- [ ] Follows best practices and coding standards
- [ ] Understandable and maintainable code

### CODE REVIEW

File: src/components/ui/accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      className={cn(
+        'flex flex-1 items-center font-medium transition-all [&[data-state=open]>svg]:rotate-180',
+        className,
+      )}
+      {...props}
+    >
+      {children}
+      <div className="flex-1" />
+      {/* display only scheduleDate is set*/}
+      {scheduleDate && (
```
Comment (KaiKoide) [inline src/components/ui/accordion.tsx:69]: The gap between the date and the icon is `6px`.
image

File: src/components/ui/accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        <p className="text-sm leading-7 text-gray-500">{scheduleDate}</p>
```
Comment (KaiKoide) [inline src/components/ui/accordion.tsx]: The font color is `#333333`.
image

File: src/components/ui/accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+   *  <Accordion type="single" collapsible className="w-full">
+        <AccordionItem value="item-1">
+          <AccordionTrigger scheduleDate={"12/31-01/07"}>Bathroom
+          <CircleEllipsis className='ml-4'/> 
+          </AccordionTrigger >
+          <AccordionContent>
+          </AccordionContent>
+        </AccordionItem>
+      </Accordion>
+   */
```
Comment (KaiKoide) [inline src/components/ui/accordion.tsx]: Could you please move the TSDoc comment above `const Accordion = Root;` so you can see the content when you hover the line?
image

**Review by KaiKoide — CHANGES_REQUESTED** [review]

**Review by KaiKoide — APPROVED** [review]
Thank you for your work! LGTM 👍

**Review by KaiKoide — COMMENTED** [review]
Could you execute `npm install` just in case?

**Comment by yuki-92c** [comment]
thanks! and execute `npm install` so please review when you have time

**Review by KaiKoide — APPROVED** [review]
Thank you! LGTM 👍

**Pull request merged**